src/sentry/rules/conditions/event_frequency.py

```diff
@@ -63,7 +63,7 @@ class EventFrequencyForm(forms.Form):
     )
     value = forms.IntegerField(widget=forms.TextInput())
     comparisonType = forms.ChoiceField(
-        choices=list(sorted(comparison_types.items(), key=lambda item: item[1])),
+        choices=comparison_types.items(),
```

Comment:
I think this still needs a `tuple(...)` call -- this is an iterable otherwise which may be exhausted